ui/index.html

```diff
@@ -23,7 +23,7 @@
 
         <!-- BODY -->
         <main id="index-container" class="oae-branding-container oae-main-content" role="main">
-            <div id="index-page-container" class="row"><!-- --></div>
+            <div id="index-page-container" tabindex="0" class="row"><!-- --></div>
```

Comment:
The examples I've found, e.g. [OSU Accessibility](http://oregonstate.edu/accessibility/skiplinks) or [WebAIM Guidelines](http://webaim.org/techniques/skipnav/) just use a standard block element as the skip link target. (They don't add the `tabindex` either but I don't see any harm in it.)

I think that screen readers might find the extra speech ("Content begins here") slightly annoying. OTOH, it would provide confirmation that the skip link worked as expected.